python/BuiltInExample1.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+
+from paraview.simple import *
+
+# create a new 'XML Image Data Reader'
+builtInExamplevti = XMLImageDataReader(FileName=['BuiltInExample1.vti'])
+builtInExamplevti.PointArrayStatus = ['Vectors_']
```

Comment:
I don't believe this line is necessary.
Thanks for double checking (e.g. with diff or paraview) that the outputs of the script are unchanged after the edit.